src/sse.ts

```diff
@@ -96,7 +96,7 @@ export const logAll = (context: Context, logLevel: string, id: string, senders?:
       return // Ignore logs without message or code.
     }
     const suffix = sender.startsWith(id) ? '' : ' ' + chalk.gray(sender)
-    const formatted = (message || code || '').replace(/\n\s*$/, '') + suffix
+    const formatted = String(message || code || '').replace(/\n\s*$/, '') + suffix
```

Comment:
Why do you need Stringify it? I thought `(message || code || '')` would be always a string, otherwise you couldn't use the replace method.